README.md

```diff
@@ -132,6 +132,12 @@ deployments. See more information in the [docker README.md file](docker/README.m
 folder includes instrucctions and configuration files to deploy GrimoireLab using
 `docker-compose` command.
 
+* Source code of the GrimoireLab components is available in `src`. Each directory is a Git submodule, so its contents will not be available after cloning the repository. To fetch all the data, you can run the following commands:
```

Comment:
Would you mind to wrap this line into a paragraph. The line is too long and each line should be ~90 characters most
so we can read it better in editors that don't split them automatically.